src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,27 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ info }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {info.title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
+      </p>
+      <UserInfo info={info} />
```

Comment:
you should pass the user here
```suggestion
      {post.user && <UserInfo user={post.user} />}
```